.github/workflows/deploy-dotorg.yml

```diff
@@ -0,0 +1,21 @@
+name: Deploy to WordPress.org
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    name: New Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: WordPress plugin deploy
+        uses: 10up/action-wordpress-plugin-deploy@stable
+        env:
+          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
```

Comment:
We'll also need to specify `SLUG: performance-lab` here, since this is the actual plugin slug, but the repo here is just named "performance". See https://github.com/10up/action-wordpress-plugin-deploy#optional-environment-variables